terragrunt/aws/glue/crawlers.tf

```diff
@@ -80,35 +80,6 @@ resource "aws_glue_crawler" "platform_support_freshdesk_production" {
   schedule = "cron(00 6 1-10 * ? *)" # 6am UTC check for schema changes on the first 10 days of each month
 }
 
-#
-# BES / CRM / Salesforce
-#
-resource "aws_glue_crawler" "bes_crm_salesforce_production" {
```

Comment:
I won’t need this crawler anymore to infer the raw schema and the number of tables. The main business logic will be handled in the ETL process.